src/test/java/com/epam/training/center/hw2/ex2/IndexTest.java

```diff
@@ -0,0 +1,94 @@
+package com.epam.training.center.hw2.ex2;
+
+import com.epam.training.center.hw2.BaseTest;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class IndexTest extends BaseTest {
+
+    private String URL = "https://jdi-testing.github.io/jdi-light/index.html";
```

Comment:
move to BaseTest to private field and remove for test classes.
Create method "openMainPage"